clients/sail_ui/lib/rpcs/bitwindow_api.dart

```diff
@@ -252,9 +263,10 @@ class BitwindowRPCLive extends BitwindowRPC {
 
 class _BitwindowAPILive implements BitwindowAPI {
   final BitwindowdServiceClient _client;
+  final BitwindowRPC api;
```

Comment:
This shouldn't be here. _BitwindowAPILive is just a raw forwarder. If you want to do more complex APIs, put them directly in `BitwindowRPC`

`BitwindowRPC -> BitwindowAPI -> BitwindowRPC -> BitwindowAPI -> ....`